parsing/parsetree.mli

```diff
@@ -174,7 +174,7 @@ and row_field_desc =
             (see 4.2 in the manual)
         *)
   | Rinherit of core_type
-        (* [ T ] *)
+        (* [ | T ] *)
```

Comment:
Since `T` is not a valid identifier for a type, maybe it would be better to update the documentation to `[ | t ] `.